### ISSUE TITLE
[Product] CMS product documentation needs updating

### ISSUE THREAD

**Issue by BerniXiongA6** [department-of-veterans-affairs/va.gov-cms#15805]
BX to come back to create details. This might be an epic if we find out there are a lot of products that need documentation.

## What you need to know
- We need to create a process for getting our products documented. There are some products that don’t have documentation and others that need updated documentation.
- BX has seen effective product documentation from previous team experiences and can identify some examples we could model and adapt for CMS
-

**Comment by EWashb** [comment]
I agree that this will probably need to be an epic! There are other product documentation tickets in the backlog too if I remember correctly.

**Comment by EWashb** [comment]
related to #12095

**Comment by maortiz-27-80** [comment]
Note: Reassigned ticket to myself 11/21. - M cc: @BerniXiongA6

**Comment by maortiz-27-80** [comment]
Unassigning the ticket from myself.  cc: @BerniXiongA6 @EWashb @michelle-dooley @gracekretschmer-metrostar
